src/cli.rs

```diff
@@ -210,43 +210,50 @@ fn build_command() -> Command<'static> {
                 .takes_value(true)
                 .value_name("UNIT")
                 .possible_values(&["millisecond", "second"])
-                .help("Set the time unit to be used. Possible values: millisecond, second."),
+                .help("Set the time unit to be used. Possible values: millisecond, second.\
+                       This option affects the standard output as well as AsciiDoc, Markdown and org-mode exports.\
```

Comment:
I would appreciate if we would "as well as all export formats except for JSON and CSV, …". Otherwise, we need to update that list whenever we add new markdown-like export formats.